### ISSUE TITLE
Persist object Label to RecipientConfiguration.Name rather than API Name

### ISSUE THREAD

**Issue by jengler74** [CodeScience/MIR3#89]
this will remove the namespace and remove underscores

i know you just put in CSS to remove the spaces but they might report on this object for some reason and the underscores would come through there. 

![image](https://cloud.githubusercontent.com/assets/7884339/15557770/19d1c768-228a-11e6-81d1-460a97cdc3b1.png)

**Comment by stevenFulgham** [comment]
Im not sure how the system will react to removing the underscores and suffix of "__c" from the configuration name. This will impact several pages as well as classes that use the name to map fields and labels using schema. I think the best thing to do is remove the underscores and suffix when you need to visualize it in a report, page, etc.

**Comment by jengler74** [comment]
That's how they currently store the Label into the Name field on RecipMapp. If you use their old tab to create one in pkg you can see the format.

![image](https://cloud.githubusercontent.com/assets/7884339/15591678/492066a4-2353-11e6-9872-61fdd4293fdb.png)

**Comment by stevenFulgham** [comment]
Ok this was a huge rework. I had the "Object Api Name" being used as the "Echo Recipient Configuration Name" I had to make adjustments in the server side code for "Recipient Mappings" and "Search Options". I also had to make changes in the "Recipient Mappings" VisualForce page javascript. I have moved my changes to Packaging as well.

![screen shot 2016-05-27 at 4 16 04 pm](https://cloud.githubusercontent.com/assets/17146094/15620200/cc6f203a-2426-11e6-9ba1-c288e236cc2b.png)
